pages/common/accelerate.md

```diff
@@ -0,0 +1,28 @@
+# Accelerate
+
+> Accelerate is a library that enables the same PyTorch code to be run across any distributed configuration by adding just four lines of code.
+> More information: <https://huggingface.co/docs/accelerate/index>.
+
+- Print environment information:
+
+`accelerate env`
+
+- Interactively set `default_config.yaml` config:
+
+`accelerate config`
+
+- Print estimated GPU memory cost of a huggingface model with different data types:
```

Comment:
```suggestion
- Print the estimated GPU memory cost of running a huggingface model with different data types:
```
Is `running` a good word here? Maybe `training` or whatever is appropriate here

Reply:
Yeah, it is intended to be an advertisement since I copied that directly from the description of the library.

I agree and will commit with your changes.

Reply:
I think `running` is appropriate since a model can be either `training` or `inference` with Accelerate.

Reply:
Okay, can I commit this suggestion then? Or can you?

Reply:
Sure, I gonna commit your suggestion , thanks.